common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -689,19 +689,19 @@ bool mon_wlan_hal_dwpal::update_stations_stats(const std::string &vap_iface_name
     size_t numOfValidArgs[10] = {0}, replyLen = strnlen(reply, HOSTAPD_TO_DWPAL_MSG_LENGTH);
     uint64_t BytesSent = 0, BytesReceived = 0, PacketsSent = 0, PacketsReceived = 0,
              LastDataDownlinkRate = 0, LastDataUplinkRate = 0, Active = 0;
-    char ShortTermRSSIAverage[32][HOSTAPD_TO_DWPAL_VALUE_STRING_LENGTH] = {'\0'};
-    char SNR[32][HOSTAPD_TO_DWPAL_VALUE_STRING_LENGTH]                  = {'\0'};
-    FieldsToParse fieldsToParse[]                                       = {
+    char ShortTermRSSIAverage[32] = {'\0'};
```

Comment:
It looks like the old code was expecting to handle up to 32 values, the new code will be limited at about 8 values:
"-77 -78 -55 -77 -78 -55 -44 -66\0" <- 32 bytes
is that intentional?